core/tests/protractor/collections.js

```diff
@@ -17,30 +17,31 @@
  */
 
 var general = require('../protractor_utils/general.js');
-var users = require('../protractor_utils/users.js');
+var UsersPage = require('../protractor_utils/UsersPage.js');
 var AdminPage = require('../protractor_utils/AdminPage.js');
 var collectionEditor = require('../protractor_utils/collectionEditor.js');
 
+var usersPage = new UsersPage.UsersPage()
 
 describe('Collections', function() {
   var adminPage = null;
 
   beforeAll(function() {
     adminPage = new AdminPage.AdminPage();
     var USERNAME = 'aliceCollections';
-    users.createUser('alice@collections.com', USERNAME);
-    users.createAndLoginAdminUser('testadm@collections.com', 'testadm');
+    usersPage.createUser('alice@collections.com', USERNAME);
```

Comment:
Please replace this string with a constant (you can decide it's name) which gets defined on line 27 next to `adminPage`.

Reply:
done